node/src/actors/chain_manager/mod.rs

```diff
@@ -1513,6 +1514,12 @@ impl ChainManager {
                     let sessions_manager_addr = SessionsManager::from_registry();
                     sessions_manager_addr.do_send(DropOutboundPeers {peers_to_drop: peers_to_unregister});
 
+                    // Include superblock beacon target in SessionsManager
+                    sessions_manager_addr.do_send(SetSuperBlockTargetBeacon {beacon: CheckpointBeacon{
+                        checkpoint: superblock_index,
+                        hash_prev_block: target_superblock_hash,
+                    }});
```

Comment:
This only sets the target superbeacon when the consensus is `SuperBlockConsensus::Different`. Should we also set it in `SuperBlockConsensus::SameAsLocal`?

Reply:
And what about the other cases? When there is no superblock consensus, should we set it to `None`, or set it to the last consolidated superblock?

Reply:
Sure!! In the other cases it should be set to None, to avoid using old target beacons